lib/LedgerLiveApiSdk.ts

```diff
@@ -153,4 +162,97 @@ export default class LedgerLiveApi {
             signedTransaction: serializeSignedTransaction(signedTransaction),
         });
     }
+
+    /**
+     * Estimate fees required to successfully broadcast a transaction.
+     * @param {string} accountId - LL id of the account
+     * @param {Transaction} transaction - the transaction to estimate
+     * 
+     * @returns {EstimatedFees} - Estimated fees for 3 level of confirmation speed
+     */
+    async estimateTransactionFees(accountId: string, transaction: Transaction): Promise<EstimatedFees> {
```

Comment:
Is this call still necessary since LL is the one computing the fees for the exchange app now?